edx/edx-ora2
# Add bulk assess view, refactor verify assessment data

**TL;DR -** Add a bulk staff assess handler to staff_assessment_mixin

JIRA: [AU-51](https://openedx.atlassian.net/browse/AU-51)

**What changed?**

- Refactor verify_assessment_parameters logic out to a separate method
- Wrote a new verify_multiple_assessment_parameters to handle a list of assessments rather than a single assessment
- Refactored almost all of staff_assess handler out to a helper, _do_staff_assess
- wrote bulk_staff_assess which calls _do_staff_assess for each staff assessment received. 
- tests

**Developer Checklist**


**Testing Instructions**

- Make an ORA
- Submit at least two submissions for the ORA
- follow instructions in https://openedx.atlassian.net/wiki/spaces/ENG/pages/3145892500/How+to+authenticate+and+query+edX+APIs+with+Postman to mke a POST as a staff userto Request URL: `https://localhost:18000/courses/<course_id>/xblock/<block _id>/handler/bulk_staff_assess`. Parameter info is included in the view docstring.
- You should receive a response, and as long as there were not any errors, the submissions should be assessed.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
